### ISSUE TITLE
Clean up directory structure

### ISSUE THREAD

**Issue by greyNearing** [NCAR/summa#150]
We should formalize the directory structure, and then modify the makefile, etc. accordingly. There should be a document about how the directory structure is organized.

**Comment by bartnijssen** [comment]
Agreed - this would be a good place to start : https://github.com/NCAR/summa/blob/master/docs/howto/summa_configuration.md#summa-directory-tree

but it is incomplete and needs some thinking through (as does all of the SUMMA documentation)

**Comment by martynpclark** [comment]
@bartnijssen -- is this now complete since you have written the documentation?
